src/Components/Quizz/StarsQuizz.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './StarsQuizz.css';
+
+const StarsQuizz = ({difficulty}) => {
+
+    const difHard = <ul><li className='etoilePleine'></li><li className='etoilePleine'></li><li className='etoilePleine'></li>
+    <li className='etoilePleine'></li><li className='etoilePleine'></li></ul>;
+    const difMedium = <ul><li className='etoilePleine'></li><li className='etoilePleine'></li><li className='etoileMi'></li>
+    <li className='etoileVide'></li><li className='etoileVide'></li></ul>
+    const difEasy = <ul><li className='etoileVide'></li><li className='etoileVide'></li><li className='etoileVide'></li>
+    <li className='etoileVide'></li><li className='etoileVide'></li></ul>
```

Comment:
It is very difficult to read and you duplicate a lot of code which is a very bad practice. Either write a loop, hard code one `ul>li` with conditional `className`s in the `return` or find another creative solution ;).

Reply:
Write english code.